windows/window_manager_plugin.cpp

```diff
@@ -465,11 +471,6 @@ void WindowManagerPlugin::HandleMethodCall(
         std::get<flutter::EncodableMap>(*method_call.arguments());
     window_manager->SetProgressBar(args);
     result->Success(flutter::EncodableValue(true));
-  } else if (method_name.compare("setIcon") == 0) {

```

Comment:
Why was this removed

Reply:
Ah, sorry this commit was pretty old that it wasn't implemented yet, and I didn't check it properly. I'll add it later.